.github/workflows/merge-queue.yml

```
@@ -1,14 +1,209 @@
 jobs:
-  success:
+  prepare:
+    runs-on: ubuntu-latest
+    timeout-minutes: 35
+    env:
+      AFFECTED_ALL: ${{ secrets.AFFECTED_ALL }}
+      SERVERSIDE_FEATURES_ON: ''
+      DOCKER_REGISTRY: 821090935708.dkr.ecr.eu-west-1.amazonaws.com/
+
+    outputs:
+      CACHE_KEYS: ${{ steps.get_cache.outputs.keys }}
```

actionlint (1.7.4)

property "get_cache" is not defined in object type {build_info: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)